pkgs/applications/misc/kepubify/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
```

Comment:
You could just have lib as an argument instead and replace the `stdenv.lib` below by `lib`

Reply:
Good point, done